css/properties/border-image.json

```diff
@@ -172,8 +175,17 @@
               "opera": {
                 "version_added": "15"
               },
+              "opera_android": {
+                "version_added": "14"
+              },
               "safari": {
-                "version_added": null
+                "version_added": "6"
+              },
+              "safari_ios": {
+                "version_added": "6"
+              },
+              "samsunginternet_android": {
+                "version_added": "1.0"
```

Comment:
WebView is missing. It should be set to `true`.

Reply:
Good catch. Fixed. Thank you!